test/pretty.test.rb

```diff
@@ -3,8 +3,8 @@
 test "objects" do
 	assert_equal_ruby Difftastic.pretty(Example.new), <<~RUBY.chomp
 		Example(
-			:@foo => 1,
-			:@bar => [2, 3, 4],
+			@foo = 1,
+			@bar = [2, 3, 4],
```

Comment:
I don't think the trailing commas here are valid Ruby. We should be able to drop them.

Reply:
In this case these are all treated as arguments as a method call, so I think you'd need them.

Reply:
Okay removed the trailing commas, we might want to remove the other trailing commands in Hash and Array at some point

Reply:
Sorry, you were right. Can we put the trailing comma back? I had this all mixed up and thought all the commas were invalid.

Reply:
Reverted them 👍🏼